book/website/communication/peer-review/peer-review-conflicts.md

```diff
@@ -0,0 +1,83 @@
+COPE Forum is a useful platform to discuss a publication ethics case with the wider scientific community with different anonymized use cases available [here](https://publicationethics.org/guidance/Case?t=conflict&classification%5B%5D=2774&sort=date). 
+We present below one of the most recent resolved cases {cite:ps}`cope2023coi`:
+
+#### Handling undisclosed peer reviewer conflict
+##### Context
+Some authors from a company recommended a peer reviewer on submitting their manuscript, who was then asked to review the manuscript. 
+This reviewer recommended acceptance without change. 
+One other reviewer recommended major revision (a methodological reviewer not a content expert) and the third reviewer recommended rejection. 
+The editor found it unusual for a review to recommend acceptance without change so looked up this reviewer and found a manuscript in press at another journal where the reviewer, as an author, disclosed relationships with the same company the authors work at. 
+These included honoraria, consultant fees, and advisory board participation. The manuscript in press indeed was about the same drug covered in the submission. 
+The reviewer answered "no" to the question asked during the peer review process on whether they had any conflicts of interest related to the research. The editor confirmed the reviewer had a valid email address.
+
+##### Question to the Forum
+What process should be undertaken when a peer reviewer does not disclose their relationships with a company, provides a suspicious review, and at the same time, where the authors (who are employed by this company) recommended that peer reviewer in the first place?
+
+##### Advice
+The presenter of the case informed the Forum that the journal had challenged both the authors and the reviewer on their failure to disclose relevant conflicts of interest. 
+The authors stated that they had not known when they recommended the reviewer that this person had links with the company. The reviewer confirmed this, and also said that they had not been involved in the work which produced the paper; thus they had felt that there was no conflict.
+The Forum agreed that the reviewer did have an interest which should have been declared. They should either have raised this with the editor, allowing the journal to make a decision on how to handle the situation, or declined to write a review.
+The editors should dismiss this review and seek a replacement if their policy is to have three reviews per paper, or go ahead with the two remaining reports. They should also provide some education for the reviewer on conflicts of interest.
+
+##### Follow-up
+The journal added a note to the paper from the editor-in-chief, stating that due to an unfortunate technical mistake when handling the article, one of the authors was also a subject editor at the same time. 
+The editor-in-chief also stated that he guaranteed that the scientific standards and honesty had not been violated in any way.
```

Comment:
This doesn't match the follow up currently on the case studies website, did it get accidentially swapped?

Here's what's on COPE's site:

> #### Follow up: 
> The editors decided that the reviewer had made an honest mistake and have closed the case.

Reply:
Fixed. Thanks!